services/core-web/src/components/Forms/incidents/IncidentForm.js

```diff
@@ -736,6 +769,68 @@ const renderInternalDocumentsComments = (props, isEditMode, handlers, parentHand
   );
 };
 
+const updateIncidentStatus = (props, isNewIncident) => {
+  const isClosed = props.incident?.status_code === "CLD";
+  const selectedStatusCode = props.formValues.status_code;
+  const responsibleInspector = props.incident?.responsible_inspector_party;
+  return !isNewIncident ? (
+    <Col span={24}>
+      <Alert
+        message={props.incidentStatusCodeHash[props.incident?.status_code] || "Undefined Status"}
+        description={
+          <Row>
+            <Col xs={24} md={18}>
+              <p>
+                {alertText(
+                  props.incident?.status_code,
+                  props.incident?.update_user,
+                  props.incident?.update_timestamp,
+                  responsibleInspector,
+                  selectedStatusCode
+                )}
+              </p>
+            </Col>
+            <Col xs={24} md={6}>
+              {props.incident?.status_code && !isClosed && (
```

Comment:
Looks like "isClosed" set above is also using props.incident?.status_code do we need this redundant check here?